[PcapPlusPlus] Unable to build sample code

**Describe the bug**
After installing pcapplusplus with `vcpkg install pcapplusplus` and copied sample code from [official doc
](pcapplusplus.github.io/docs/tutorials/intro#writing-a-simple-app-including-a-makefile) I get linking error from PcapPlusPlus libs.

**Environment**
- OS: Windows and Linux
- Compiler: clang | g++

**To Reproduce**
Steps to reproduce the behavior:
1. ./vcpkg install pcapplusplus
2. Add following row in a cmake file:
2.a find_package(unofficial-pcapplusplus CONFIG REQUIRED)
2.b target_link_libraries(${PROJECT_NAME} ${Pcap_LIB} unofficial::pcapplusplus::pcappp unofficial::pcapplusplus::commonpp unofficial::pcapplusplus::packetpp )
3. Try build main.cpp and see linking error

**Expected behavior**
It should build and run without any problem
 
**Failure logs**
```
[build] LINK: command "C:\PROGRA~1\MICROS~4\2022\COMMUN~1\VC\Tools\MSVC\1434~1.319\bin\Hostx86\x64\link.exe /nologo CMakeFiles\pcappp.dir\main.cpp.obj /out:pcappp.exe /implib:pcappp.lib /pdb:pcappp.pdb /version:0.0 /machine:x64 /INCREMENTAL:NO /subsystem:console C:\vcpkg\installed\x64-windows\lib\Pcap++.lib C:\vcpkg\installed\x64-windows\lib\Common++.lib C:\vcpkg\installed\x64-windows\lib\Packet++.lib C:\vcpkg\installed\x64-windows\lib\Common++.lib C:\vcpkg\installed\x64-windows\lib\wpcap.lib C:\vcpkg\installed\x64-windows\lib\Packet.lib ws2_32.lib iphlpapi.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:pcappp.exe.manifest" failed (exit code 1120) with the following output:
[build] Pcap++.lib(PcapFileDevice.cpp.obj) : error LNK2019: unresolved external symbol light_pcapng_open_read referenced in function "public: virtual bool __cdecl pcpp::PcapNgFileReaderDevice::open(void)" (?open@PcapNgFileReaderDevice@pcpp@@UEAA_NXZ)
[build] Pcap++.lib(PcapFileDevice.cpp.obj) : error LNK2019: unresolved external symbol light_pcapng_open_write referenced in function "public: bool __cdecl pcpp::PcapNgFileWriterDevice::open(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?open@PcapNgFileWriterDevice@pcpp@@QEAA_NAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@000@Z)
[build] Pcap++.lib(PcapFileDevice.cpp.obj) : error LNK2019: unresolved external symbol light_pcapng_open_append referenced in function "public: virtual bool __cdecl pcpp::PcapNgFileWriterDevice::open(bool)" (?open@PcapNgFileWriterDevice@pcpp@@UEAA_N_N@Z)
[build] Pcap++.lib(PcapFileDevice.cpp.obj) : error LNK2019: unresolved external symbol light_create_default_file_info referenced in function "public: virtual bool __cdecl pcpp::PcapNgFileWriterDevice::open(void)" (?open@PcapNgFileWriterDevice@pcpp@@UEAA_NXZ)
[build] Pcap++.lib(PcapFileDevice.cpp.obj) : error LNK2019: unresolved external symbol light_create_file_info referenced in function "public: bool __cdecl pcpp::PcapNgFileWriterDevice::open(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?open@PcapNgFileWriterDevice@pcpp@@QEAA_NAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@000@Z)
[build] Pcap++.lib(PcapFileDevice.cpp.obj) : error LNK2019: unresolved external symbol light_free_file_info referenced in function "public: bool __cdecl pcpp::PcapNgFileWriterDevice::open(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?open@PcapNgFileWriterDevice@pcpp@@QEAA_NAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@000@Z)
[build] Pcap++.lib(PcapFileDevice.cpp.obj) : error LNK2019: unresolved external symbol light_pcang_get_file_info referenced in function "public: class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl pcpp::PcapNgFileReaderDevice::getCaptureApplication(void)const " (?getCaptureApplication@PcapNgFileReaderDevice@pcpp@@QEBA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@XZ)
[build] Pcap++.lib(PcapFileDevice.cpp.obj) : error LNK2019: unresolved external symbol light_get_next_packet referenced in function "public: bool __cdecl pcpp::PcapNgFileReaderDevice::getNextPacket(class pcpp::RawPacket &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > &)" (?getNextPacket@PcapNgFileReaderDevice@pcpp@@QEAA_NAEAVRawPacket@2@AEAV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z)
[build] Pcap++.lib(PcapFileDevice.cpp.obj) : error LNK2019: unresolved external symbol light_write_packet referenced in function "public: bool __cdecl pcpp::PcapNgFileWriterDevice::writePacket(class pcpp::RawPacket const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?writePacket@PcapNgFileWriterDevice@pcpp@@QEAA_NAEBVRawPacket@2@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z)
[build] Pcap++.lib(PcapFileDevice.cpp.obj) : error LNK2019: unresolved external symbol light_pcapng_close referenced in function "public: virtual void __cdecl pcpp::PcapNgFileReaderDevice::close(void)" (?close@PcapNgFileReaderDevice@pcpp@@UEAAXXZ)
```

--- a/ports/pcapplusplus/CMakeLists.txt
+++ b/ports/pcapplusplus/CMakeLists.txt
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.12)
 
-project(pcapplusplus CXX)
+project(pcapplusplus C CXX)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
 
@@ -67,9 +67,9 @@ endif()
 # pcap++
 file(GLOB PCAPPP_HEADERS "${CMAKE_CURRENT_LIST_DIR}/Pcap++/header/*.h")
 file(GLOB PCAPPP_SOURCES "${CMAKE_CURRENT_LIST_DIR}/Pcap++/src/*.cpp")
-file(GLOB LIGHTPCAPNG_SOURCES "${CMAKE_CURRENT_LIST_DIR}/3rdParty/LightPcapNg/LightPcapNg/src/*.cpp")
+file(GLOB LIGHTPCAPNG_SOURCES "${CMAKE_CURRENT_LIST_DIR}/3rdParty/LightPcapNg/LightPcapNg/src/*.c")
 
-add_library(pcappp ${PCAPPP_SOURCES})
+add_library(pcappp ${PCAPPP_SOURCES} ${LIGHTPCAPNG_SOURCES})
 
 target_include_directories(pcappp PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/Pcap++/header> $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/3rdParty/LightPcapNg/LightPcapNg/include> "${PCAP_INCLUDES}" $<INSTALL_INTERFACE:include>)
 

--- a/ports/pcapplusplus/vcpkg.json
+++ b/ports/pcapplusplus/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "pcapplusplus",
   "version-string": "22.05",
-  "port-version": 1,
+  "port-version": 2,
   "description": "PcapPlusPlus is a multi-platform C++ library for capturing, parsing and crafting of network packets",
   "homepage": "https://github.com/seladb/PcapPlusPlus",
   "license": null,

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5730,7 +5730,7 @@
     },
     "pcapplusplus": {
       "baseline": "22.05",
-      "port-version": 1
+      "port-version": 2
     },
     "pcg": {
       "baseline": "2021-04-06",

--- a/versions/p-/pcapplusplus.json
+++ b/versions/p-/pcapplusplus.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "8e13cef16f2b535aabf5672b3617ff0559772f61",
+      "version-string": "22.05",
+      "port-version": 2
+    },
+    {
       "git-tree": "fc809c85d5fb8f4c3132cac73cd48df0704dd170",
       "version-string": "22.05",
       "port-version": 1

